components/elm/src/biogeochem/NitrogenStateUpdate2Mod.F90

```diff
@@ -31,37 +31,33 @@ module NitrogenStateUpdate2Mod
 contains
 
   !-----------------------------------------------------------------------
-  subroutine NitrogenStateUpdate2(num_soilc, filter_soilc, num_soilp, filter_soilp, &
-       nitrogenflux_vars, nitrogenstate_vars)
+  subroutine NitrogenStateUpdate2(num_soilc, filter_soilc, num_soilp, filter_soilp,dt)
     !
     ! !DESCRIPTION:
     ! On the radiation time step, update all the prognostic nitrogen state
     ! variables affected by gap-phase mortality fluxes
     ! NOTE - associate statements have been removed where there are
     ! no science equations. This increases readability and maintainability
     !
-    use tracer_varcon, only : is_active_betr_bgc      
+      !$acc routine seq
+    use tracer_varcon, only : is_active_betr_bgc
     ! !ARGUMENTS:
     integer                  , intent(in)    :: num_soilc       ! number of soil columns in filter
     integer                  , intent(in)    :: filter_soilc(:) ! filter for soil columns
     integer                  , intent(in)    :: num_soilp       ! number of soil patches in filter
     integer                  , intent(in)    :: filter_soilp(:) ! filter for soil patches
-    type(nitrogenflux_type)  , intent(inout) :: nitrogenflux_vars
-    type(nitrogenstate_type) , intent(inout) :: nitrogenstate_vars
+    !type(nitrogenflux_type)  , intent(inout) :: nitrogenflux_vars
+    !type(nitrogenstate_type) , intent(inout) :: nitrogenstate_vars
+    real(r8) , intent(in)   :: dt      ! radiation time step (seconds)
+
     !
     ! !LOCAL VARIABLES:
     integer  :: c,p,j,l ! indices
     integer  :: fp,fc   ! lake filter indices
-    real(r8) :: dt      ! radiation time step (seconds)
     !-----------------------------------------------------------------------
 
-    associate(                      & 
-         nf => nitrogenflux_vars  , &
-         ns => nitrogenstate_vars   &
-         )
-
       ! set time steps
-      dt = real( get_step_size(), r8 )
+      !#py dt = real( get_step_size(), r8 )
```

Comment:
There are too many lines that are commented out. I'm now not going to add a comment on each line hat is commented out in this first review.